documentation/source/install_devel.rst

```diff
@@ -0,0 +1,17 @@
+.. _install-devel:
+
+**********************************
+Installing the Development Version
+**********************************
+
+To install the development version of cocotb, run:
+
+.. code-block:: bash
+
+    pip install https://github.com/cocotb/cocotb/archive/master.zip
```

Comment:
I'd probably recommend the git install, `pip install git+https://github.com/cocotb/cocotb.git` or similar.

Reply:
Although if users do not have git, the zip install is a nice backup option

Reply:
Let's go for the safe option and point to the git capabilities of pip under a Key Topics section.